AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoConstants.h

```diff
@@ -314,3 +314,20 @@ extern NSString * const kAlfrescoWorkflowVariableTaskInvitePendingOutcome;
 extern NSString * const kAlfrescoWorkflowTaskTransitionApprove;
 extern NSString * const kAlfrescoWorkflowTaskTransitionAccept;
 extern NSString * const kAlfrescoWorkflowTaskTransitionReject;
+
+/**---------------------------------------------------------------------------------------
+ * @name Connection Diagnostic Constants
+ --------------------------------------------------------------------------------------- */
+extern NSString * const kAlfrescoConfigurationDiagnosticDidStartEventNotification;
+extern NSString * const kAlfrescoConfigurationDiagnosticDidEndEventNotification;
+
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsLoading;
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsSuccess;
```

Comment:
In the end both methods accomplish the same thing, the only difference being how the equality of two events is determined. I would choose NS_ENUM. if there are no more reasons to not choose it please add them here. I will start to implement the necessary changes.